source/2022.tex

```diff
@@ -0,0 +1,95 @@
+		\begin{enumerate}
+			\item Organising, presenting and promoting Python related conferences, meetings and events at venues within the UK,
+			\item Supporting and promoting Python related projects and events organised by UKPA members, and,
+			\item Supporting UKPA members wishing to attend Python related conferences, meetings and events.
+		\end{enumerate}
+		\item To advance education for the public benefit in the use and understanding of the Python programming language worldwide by supporting and promoting overseas Python related conferences, meetings and events.
+	\end{enumerate}
+
+    The primary activity of the UKPA is to organise and run the annual PyCon UK conference.
+
+    \section{Structure, Governance and Management}
+
+        \subsection{Governing Document}
+        The UKPA's governing document is its constitution, the current version of which is available at https://github.com/PyconUK/ukpa-constitution/releases/latest.
+
+        \subsection{UKPA Members}
+        The UKPA membership is the electoral body to which the trustees are accountable.
+		There are two classes of membership:
+
+		\begin{enumerate}
+			\item Members: have the full rights and obligations described in the UKPA constitution, including the right to vote on members decisions.
+			\item Associate Members: non-voting class of membership as described in the UKPA constitution.
+		\end{enumerate}
+
+        \subsection{Trustee Appointment}
+		At the AGM in 2021, Tania Allard and Cecilia Liao retired as a trustees in accordance with section 13.1.1 of the UKPA's constitution.
+		Tania Allard was elected as a trustee for a further term in accordance with section 13.1.4 of the UKPA's constitution.
+
+	    \subsection{Risk and Internal Control}
+        The trustees have identified the major risks to which they believe the UKPA is exposed. These are:
+
+          \subsubsection{Reduction or Loss of Members}
+            If there were a reduction in membership of the UKPA then there would have to be a contraction, consolidation or closure of its activities.
+
+            \subsubsection{Injury to Members, Conference Attendees or Trustees}
+            The UKPA requires insurance to cover this potential risk and the trustees intend to put that insurance in place once the UKPA has sufficient funds to cover the cost of the premium.
+
+            \subsubsection{Reduction or Loss of Volunteers}
+            The UKPA is totally reliant upon volunteers to run and administer the activities of the UKPA. If there were a reduction in the number of volunteers to an unacceptable level, then there would have to be a contraction, consolidation or closure of activities.
+
+    \section{Achievements and Performance}
+    	As a result of the ongoing COVID19 pandemic, the trustees decided to cancel the 2021 PyCon UK event and there was no other significant activity during the course of the year.
```

Comment:
nitpick / suggestion: Most style guides seem to use `COVID-19`; I've not really seen `COVID19` in any usage